06-Plotting-Time-Series-Rasters-in-R.Rmd

```diff
@@ -229,15 +241,34 @@ we adjust the layout to be a matrix of 4 columns and 4 rows.
 ```{r adjust-layout }
 #use level plot to create a nice plot with one legend and a 4x4 layout.
```

Comment:
Changed for consistency in previous, this, and challenge chunk.  Different arrangements in each to teach/ reinforce how it is changed.